source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -1,164 +1,163 @@
+    When you delegate, the CCDs do not leave your wallet. They are locked there, so you can not spend them, and their value is added to the value of the staking pool or to :term:`passive delegation`. But the CCDs never leave your wallet, so they can never be lost. There is no more risk delegating than not delegating. The worst thing that can happen is that the staking pool stops producing blocks but stays open, and then you will not get any rewards. If the pool closes, your stake will be transferred to passive delegation automatically and from here you can easily stop delegation if you want to. It is important to use `CCDScan`_ to monitor validator and pool performance.
 
-.. dropdown:: Can I delegate to more than one baker at a time?
+.. dropdown:: Can I delegate to more than one validator at a time?
 
-    Yes. You can delegate to multiple bakers but an account can only have one delegation. To delegate to multiple bakers, you need multiple accounts.
+    Yes. You can delegate to multiple validators but an account can only have one delegation. To delegate to multiple validators, you need multiple accounts.
 
-.. dropdown:: Can I move from one baker pool to another without having to wait for the next pay day?
+.. dropdown:: Can I move from one staking pool to another without having to wait for the next pay day?
 
-    Changes to the pools take effect every 24 hours. So increasing the stake, moving the stake between pools or between :term:`passive delegation` and a baking pool all take effect at the next :term:`pay day`.
+    Changes to the pools take effect every 24 hours. So increasing the stake, moving the stake between pools or between :term:`passive delegation` and a staking pool all take effect at the next :term:`pay day`.
 
 .. dropdown:: How much does it cost to begin delegating?
 
     There is no minimum amount required to delegate stake to a pool or enable :term:`passive delegation`. You just have to have enough CCD left in your disposable balance to cover transaction fees.
 
 .. dropdown:: How much can I expect to earn as a delegator?
 
-    This depends on how much you delegate, the baker pool you select and how reliable this baker is. It also depends on how much stake the baker has. It is important to use `CCDScan`_ to research bakers before delegating stake.
+    This depends on how much you delegate, the staking pool you select and how reliable this validator is. It also depends on how much stake the validator has. It is important to use `CCDScan`_ to research validators before delegating stake.
```

Comment:
This depends on how much you delegate, the staking pool you select, **the delegation commissions set by that pool's owner** and how reliable this validator is.